auth-web/src/components/auth/manage-business/BusinessLookup.vue

```diff
@@ -0,0 +1,179 @@
+        <v-row class="business-lookup-result pt-1">
+          <v-col cols="2" class="result-identifier">{{item.identifier}}</v-col>
+          <v-col cols="8" class="result-name">{{item.name}}</v-col>
+          <v-col cols="2" class="result-action">
+            <span v-if="item.disabled" class="added">Added</span>
+            <span v-else class="select">Select</span>
+          </v-col>
+        </v-row>
+      </template>
+    </v-autocomplete>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Emit, Watch } from 'vue-property-decorator'
+import { BusinessLookupResultIF } from '@/models'
+import BusinessLookupServices from '@/services/business-lookup.services'
+import { Debounce } from 'vue-debounce-decorator'
+import Vue from 'vue'
+import { namespace } from 'vuex-class'
+
+const BusinessModule = namespace('business')
+
+enum States {
+  INITIAL = 'initial',
+  SEARCHING = 'searching',
+  SHOW_RESULTS = 'show results',
+  NO_RESULTS = 'no results',
+  SUMMARY = 'summary'
+}
+
+/*
+ * See PPR's BusinessSearchAutocomplete.vue for a Composition API example.
+ */
+@Component({})
+export default class BusinessLookup extends Vue {
+  // enum for template
+  readonly States = States
+
+  // action from business module
+  @BusinessModule.Action('isAffiliated')
+  private readonly isAffiliated!: (identifier: string) => Promise<boolean>
+
+  /** V-model for search field. */
+  searchField = ''
+
+  /** Results from business lookup API. */
+  searchResults = [] as Array<BusinessLookupResultIF>
+
+  /** State of this component. */
+  state = States.INITIAL
+
+  /** Called when searchField property has changed. */
+  @Watch('searchField')
+  @Debounce(600)
+  private async onSearchFieldChanged (): Promise<void> {
+    // safety check
+    if (this.searchField && this.searchField.length > 2) {
+      this.state = States.SEARCHING
+      const searchStatus = null // search all (ACTIVE + HISTORICAL)
+      this.searchResults = await BusinessLookupServices.search(this.searchField, searchStatus).catch(() => [])
+
+      // enable or disable items according to whether they have already been added
+      for (const result of this.searchResults) {
+        result.disabled = await this.isAffiliated(result.identifier)
+      }
+
+      // display appropriate section
+      this.state = (this.searchResults.length > 0) ? States.SHOW_RESULTS : States.NO_RESULTS
+    } else {
+      // reset variables
+      this.searchResults = []
+      this.state = States.INITIAL
+    }
+  }
+
+  /** When an item has been selected, emits event with business object. */
+  @Emit('business')
+  onItemSelected (input: BusinessLookupResultIF): void {
+    // safety check
+    if (input) {
+      // change to summary state
+      this.state = States.SUMMARY
+    }
+    // event data is automatically returned
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+@import '@/assets/styles/theme.scss';
+
+.font-size-12 {
+  font-size: 12px;
```

Comment:
This is the only class of its kind so I didn't put it in base.scss. If we get more of these then we should move them.